.gitignore

```diff
@@ -9,4 +9,8 @@ test2.py
 **/*.mat
 *_dir.txt
 estimatestest.py
-nwbdemotest.py
\ No newline at end of file
+nwbdemotest.py
+nwbextractor_SI.py
+ophys.py
+ecephys.py
+scnitzerTraceExtractor.py
```

Comment:
These are odd things to add to a `.gitignore` file. If you are downloading tutorials, just put them in a separate directory/project. It seems like the scnitzerTraceEctractor.py (which is still misspelled) should be in this project though